app/i18n/es.json

```diff
@@ -682,7 +682,8 @@
   "errors": {
     "generic": "Hubo un error inesperado. Por favor intente más tarde\n",
     "invalidEmail": "Correo inválido",
-    "invalidPhoneNumber": "Esto no es un número de teléfono válido"
+    "invalidPhoneNumber": "Esto no es un número de teléfono válido",
+    "tooManyRequestsPhoneCode": "Espere 60 minutos antes de solicitar otro mensaje de texto"
```

Comment:
Would be good to get a native Spanish speaker to review this before merge.

Reply:
@dolcalmi